pkgs/top-level/perl-packages.nix

```diff
@@ -12389,7 +12389,7 @@ let
       export PERL_MB_OPT="--install_base=$out --prefix=$out"
       substituteInPlace Po4aBuilder.pm --replace "\$self->install_sets(\$self->installdirs)->{'bindoc'}" "'$out/share/man/man1'"
     '';
-    buildPhase = "perl Build.PL --install_base=$out; ./Build build";
+    buildPhase = "perl Build.PL --install_base=$out --install_path=\"sbin=$out/bin\" --install_path=\"lib=$out/lib/perl5/site_perl\"; ./Build build ";
```

Comment:
Why a this needed?

Is not there something like `python.sitePackages` for Perl so the path does not need to be specified manually?

Reply:
I don't know what `python.sitePackages` is. Without this change the package was broken - whenever you try to run it's executables the app can't find it's own packages. I copied this buildPhase from another perl package and it fixed that issue.

Reply:
Try `perl.libPrefix`, see https://github.com/NixOS/nixpkgs/blob/b547594fa67cd667678b628093ca031b941a3457/pkgs/development/interpreters/perl/default.nix#L105

Reply:
fixed